porcupine/plugins/run/no_terminal.py

```diff
@@ -316,7 +349,11 @@ def toggle_visible(junk_event: object = ...) -> None:
 
     runner.hide_button.bind("<Button-1>", toggle_visible, add=True)
     runner.stop_button.bind("<Button-1>", runner.stop_executor, add=True)
+    runner.pause_button.bind("<Button-1>", runner.pause_resume_executor, add=True)
     menubar.get_menu("Run").add_command(label="Show/hide output", command=toggle_visible)
+    menubar.get_menu("Run").add_command(
+        label="Pause/resume process", command=runner.pause_resume_executor
```

Comment:
On Windows, this should be either grayed out or not exist at all. The button is already invisible because it isn't packed.